tests/e2e/webpack.config.js

```diff
@@ -32,6 +32,12 @@ module.exports = {
             loader: 'cypress-cucumber-preprocessor/lib/featuresLoader'
           }
         ]
+      },
+      {
+        test: /@empathyco\/x-components/,
+        exclude: /\.css$/,
+        enforce: 'pre',
+        use: ['babel-loader']
       }
```

Comment:
If I'm not mistaken you can simplify this config. This is only needed to transpile the `forEach` utility which is imported by the support file. There is no CSS imported, but just in case I would add the `.js` extension at the end of the regex, as it is the kind of files that babel has to transpile. Also you don't need to make this a pre-loader, a normal loader is fine.

```suggestion
      {
        test: /@empathyco\/x-components.+\.js$/,
        use: ['babel-loader']
      },```